cmd/util.go

```diff
@@ -385,26 +385,21 @@ func shouldAuditCSC(podSpec PodSpecV1, container ContainerV1) bool {
 	return false
 }
 
```

Comment:
Code changes look good, I would suggest we rename variable `exist` to `labelExists` for more verbosity

Reply:
Done